grid-demo/src/app/angular/advanced-row-selection/advanced-selection.directive.ts

```diff
@@ -0,0 +1,39 @@
+import { Directive, ElementRef, HostListener } from '@angular/core';
+import { ClrDatagrid } from '@clr/angular';
+
+const clrRowTagName = 'CLR-DG-ROW';
```

Comment:
This could be moved into the `hasDatagridRowParent` function because it's only used there.

Reply:
agreed. moved